streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/FlinkCluster.java

```diff
@@ -138,23 +144,34 @@ public URI getActiveAddress() {
         return null;
     }
 
-    public boolean verifyConnection() {
-        if (address == null) {
-            return false;
-        }
-        String[] array = address.split(",");
-        for (String url : array) {
-            try {
-                new URI(url);
-            } catch (Exception ignored) {
+    public boolean verifyClusterConnection() {
+        if (ExecutionMode.REMOTE.equals(this.getExecutionModeEnum()) ||
+            ExecutionMode.YARN_SESSION.equals(this.getExecutionModeEnum())) {
+            if (address == null) {
                 return false;
             }
-            try {
-                HttpClientUtils.httpGetRequest(url, RequestConfig.custom().setConnectTimeout(2000).build());
-                return true;
-            } catch (Exception ignored) {
-                //
+            String[] array = address.split(",");
+            for (String url : array) {
```

Comment:
Why is there a cycle here?

Reply:
> Why is there a cycle here?

For example,   the address may be a set of addresses, such as: 192.168.0.1, 192.168.0.2, 192.168.0.3

Reply:
Same question: Isn't the address JM url? 

If the address is JM url, why is it a list?

Reply:
> Same question: Isn't the address JM url?
> 
> If the address is JM url, why is it a list?

e.g:
1): cluster is  flink standalone cluster, is a set of addresses
2): cluster is yarn-session cluster, is yarn HA addresses

Reply:
As I understand, yarn session cluster is a yarn job. It just has a JobManager and when old JM is crashed, yarn will start a new JM, right?

If so, I don't understand the HA address.

Reply:
> As I understand, yarn session cluster is a yarn job. It just has a JobManager and when old JM is crashed, yarn will start a new JM, right?
> 
> If so, I don't understand the HA address.

I don't want to discuss this part too much, If you set a single url here, corresponding to a unit of yarn node, once the node hangs, the subsequent can not do other work.

Reply:
>  It just has a JobManager and when old JM is crashed, yarn will start a new JM, right?

As I understand, yarn will start a new JM after old JM crashed. So I wonder how to get the HA address. Maybe I missed some background.